lib/MiqVm/MiqRhevmVm.rb

```diff
@@ -177,4 +181,11 @@ def unmount_storage
       $log.warn "#{log_header} Failed to unmount all items from <#{nfs_mount_root}>.  Reason: <#{$!}>"
     end
   end
+
+  private
+
+  # Output attributes of LV in column format and parse the 4th column to retrieve active status
+  def lv_active?(lv_path)
+    `sudo lvdisplay #{lv_path} -c | cut -d ":" -f4`.gsub("\n","") == "0" ? false : true
```

Comment:
Let me check if there's a more programmatic way of checking this, but could you do `lvdisplay /dev/vg_system/lv_os | grep 'LV Status'|awk '{print $3}'`

e.g.
```
# lvdisplay /dev/vg_system/lv_os | grep 'LV Status'
  LV Status              available
# lvdisplay /dev/vg_system/lv_os | grep 'LV Status'|awk '{print $3}'
available
```

Reply:
Per https://saratahosting.com/manpages/lvdisplay.8.html you can use `--colon` to print a more structured result which should be more reliable for machine parsing 
```
# lvdisplay --colon /dev/vg_system/lv_os 
  /dev/vg_system/lv_os:vg_system:3:1:-1:1:22020096:2688:-1:0:-1:253:0
```

Or lvs can take an explicit output column list:
```
# lvs -o lv_active /dev/vg_system/lv_os 
  Active
  active
```

I assume it is column title then value

Reply:
On a test LV I created:
```
[manageiq@nasark vmdb]$ sudo lvdisplay /dev/vg_test/lv_test | grep 'LV Status'|awk '{print $3}'
available
[manageiq@nasark vmdb]$ sudo lvchange -an /dev/vg_test/lv_test
[manageiq@nasark vmdb]$ sudo lvdisplay /dev/vg_test/lv_test | grep 'LV Status'|awk '{print $3}'
NOT
```

Reply:
When not active:
```
  --- Logical volume ---
  LV Path                /dev/vg_test/lv_test
  LV Name                lv_test
  VG Name                vg_test
  LV UUID               uuid
  LV Write Access        read/write
  LV Creation host, time , 2024-06-11 14:44:24 -0400
  LV Status              NOT available
  LV Size                800.00 MiB
  Current LE             50
  Segments               1
  Allocation             inherit
  Read ahead sectors     auto

```

Reply:
> Per https://saratahosting.com/manpages/lvdisplay.8.html you can use --colon to print a more structured result which should be more reliable for machine parsing

Yes the `-c` I'm using in the command is the same as `--colon`. The `lvs` option does look cleaner but returns empty when the lv is inactive. Let me know what you prefer
```
[root@nasark vmdb]# lvs --noheadings -o lv_active /dev/vg_test/lv_test 
        
```

Reply:
with lvs we could do:
```
def lv_active?(lv_path)
    `sudo lvs --noheadings -o lv_active #{lv_path}`  == "active" ? true : false
end
```

Reply:
Yeah I think the `lvs` way looks pretty clean